tests/README.md

```diff
@@ -104,6 +104,8 @@ Some helpful commands:
 
 The tests need ```/dev/kvm``` to be accessible to non-root users on each node:
 
+    $ sudo modprobe kvm
+    $ printf 'kvm\n' | sudo tee /etc/modules-load.d/kvm.conf
```

Comment:
A simple "echo kvm" would do, but as this usually gets copy&paste'd, it makes little difference.